### ISSUE TITLE
world seed desync

### ISSUE THREAD

**Issue by jevry** [IntQuant/noita_entangled_worlds#67]
didn't see the issue anywhere else yet. it's possible for the client and host to have different seeds. for us it triggered i think because we launched the proxy before the game.
when this happens the client experiences chunks of ground spawning in, perfectly vertical and horizontal lines of ground not existing entirely(as if a 5 pixel wide laser with infinite range and penetration was shot), chunks of ground despawning, the host can move through ground only the client can see (as expected) and vice versa. also maximum screen shake as if a nuke is going off offscreen constantly.

in short, the proxy is trying to merge the 2 seeds into one and causing loads of bugs because of it.

**Comment by bgkillas** [comment]
has not happened for a while, i think
